xds/internal/balancer/edsbalancer/config.go

```diff
@@ -35,8 +35,9 @@ type EDSConfig struct {
 	// FallBackPolicy represents the load balancing config for the
 	// fallback.
 	FallBackPolicy *loadBalancingConfig
-	// Name to use in EDS query.  If not present, defaults to the server
-	// name from the target URI.
+	// Cluster name.
+	ClusterName string
+	// Name to use in EDS query. If not set, use ClusterName.
```

Comment:
Super nit: These comments should also follow the Go documentation style where it starts with the field name.